packages/react-renderer/README.md

```diff
@@ -473,6 +473,145 @@ module.exports = {
 };
 ```
 
+### Placeholder images for Next.js Image component
+
+For low quality image placeholders (LQIP) we can use [Plaiceholder](https://github.com/joe-bell/plaiceholder). Plaiceholder will generate base64 encoded images which we pass to `next/image` as the [`blurDataUrl`](https://nextjs.org/docs/api-reference/next/image#blurdataurl) prop. In this example we'll query a rich text field and generate a placeholder image for each embedded asset.
+
+First, install Plaiceholder:
+
+```sh
+# npm
+npm i plaiceholder
+
+# Yarn
+yarn add plaiceholder
+```
+
+Note that Plaiceholder uses `sharp` under the hood, but as `next/image` ships with it, we don't need to install it separately.
+
+Here's a full blown example for a single blog post page with rich text content.
+
+```js
+// [slug.jsx]
+import { RichText } from '@graphcms/rich-text-react-renderer';
+import { getPlaiceholder } from 'plaiceholder';
+import { fetchFromGraphCMS } from '../../lib/graphcms';
+import Image from 'next/image';
+
+// Page template
+const SinglePostPage = ({ data }) => {
+  const { title, description, content } = data;
+
+  return (
+    <>
+      {/* ... */}
+      <RichText
+        content={content.json}
+        references={content.references}
+        renderers={{
+          Asset: {
+            image: ({ url, alt, caption, width, height, blurDataUrl }) => {
+              return (
+                <Image
+                  src={url}
+                  alt={alt}
+                  width={width}
+                  height={height}
+                  placeholder={blurDataUrl ? 'blur' : 'empty'}
+                  blurDataURL={blurDataUrl}
+                />
+              );
+            },
+          },
+        }}
+      />
+    </>
+  );
+};
+
+// Relevant paths from CMS to Next
+export const getStaticPaths = async () => {
+  const data = await fetchFromGraphCMS({
+    query: `
+      {
+        posts {
+          slug
+        }
+      }`,
+    variables: {},
+    preview: true,
+  });
+
+  const postPaths = data.posts.map((item) => {
+    return {
+      params: {
+        slug: item.slug,
+      },
+    };
+  });
+
+  return {
+    paths: postPaths,
+    fallback: false,
+  };
+};
```

Comment:
I think we can remove this part. Let's show only the `getStaticProps` function to keep it straightforward.

Reply:
Thanks @jpedroschmitz, sounds good to me. Would it be ok to leave a placeholder reminder of the `getStaticPaths` like this:

```suggestion
export const getStaticPaths = async () => {
  // Get your paths here.
};
```

That would make the example complete (and rookie-friendly) without adding that many lines.

Reply:
Yeah, sounds perfect!

Reply:
Great, change pushed!